locations/categories.py

```diff
@@ -825,21 +825,32 @@ class Clothes(Enum):
     WOMEN = "women"
 
 
-def apply_clothes(clothes: [Clothes], item: Feature):
+def apply_clothes(clothes: type[Clothes] | list[type[Clothes]], item: Feature | dict) -> None:
     """
-    Apply clothing categories to a Feature. If the Feature already has
+    Apply clothing categories to a feature. If the feature already has
     clothing categories defined, this function will append to the list of
     clothing categories rather than overwriting existing clothing categories.
     When appending, the list of clothing categories is sorted and then each
     value is separated with a semi-colon. Duplication of clothing categories
     is avoided by ignoring subsequent attempts to add an already existing
     clothing category.
-    :param clothes: array of Clothes Enum members
-    :param item: Feature which should have clothing categories applied.
+
+    :param clothes: single Clothes Enum member or array of Clothes Enum members.
+    :param item: feature which should have clothing categories applied.
     """
-    for c in clothes:
-        apply_yes_no(f"clothes:{c.value}", item, True)
-        apply_category({"clothes": c.value}, item)
+    if item["extras"].get("clothes"):
+        current = item["extras"]["clothes"].split(";")
+    else:
+        current = []
+
+    for v in clothes if isinstance(clothes, list) else [clothes]:
+        if v.value not in current:
+            current.append(v.value)
+            apply_yes_no(f"clothes:{v.value}", item, True)
+
+    current.sort()
```

Comment:
Thanks for removing apply_category.

I'm not sure we should be doing sort here (and in similar functions) though. In OSM these should be ordered by importance to the POI. Nobody cares about `clothes`, so I'm not gonna cry, but this wouldn't fly for `cuisine`.